node/resolvers/index.ts

```diff
@@ -1760,11 +1760,55 @@ export const resolvers = {
       try {
         const paymentRules = await payments.rules()
 
-        const promissoryConnectors = paymentRules.filter(
-          rule => rule.connector.implementation === CONNECTOR.PROMISSORY
+        const enabledConnectors = paymentRules.filter(
+          rule => rule.enabled === true
         )
 
-        return promissoryConnectors.map(connector => connector.paymentSystem)
+        const enabledPaymentSystems = enabledConnectors.map(
+          connector => connector.paymentSystem
+        )
+
+        const uniquePaymentSystems = enabledPaymentSystems.filter(
+          (value, index, self) => {
+            return (
+              index ===
+              self.findIndex(t => t.id === value.id && t.name === value.name)
+            )
+          }
+        )
+
+        const uniquePaymentSystemsWithoutCreditCards = uniquePaymentSystems.filter(
+          value => {
+            const creditCards = [
+              'Visa',
+              'Mastercard',
+              'Diners',
+              'American Express',
+              'Hipercard',
+              'Discover',
+              'Aura',
+              'Elo',
+              'Banricompras',
+              'JCB',
+              'Cabal',
+              'Nativa',
+              'Naranja',
+              'Nevada',
+              'Shopping',
+              'Credz',
+            ]
```

Comment:
I would put this constant array outside the filter function to avoid processing each loop and make it clear. However, The best solution would be if this array comes from the app setting, what do you think?